### DISCUSSION TITLE
How can I modify the value of a TextEntry?

### DISCUSSION THREAD

**Discussion by NgYueHong** [filamentphp/filament#12647] [Help]
### Package

Infolist builder

### Package Version

3.2

### How can we help you?

The following code will return something like `MYR 23.00`. but what I want is  `RM 23.00`. So, I have to do it the manual way like `'RM ' . number_format($value, 2)`. But how can I modify the value of a TextEntry?
```
TextEntry::make('price')->money('myr')
```

**Comment by leandrocfe** [comment]
>formatStateUsing()

https://filamentphp.com/docs/3.x/infolists/entries/text/#custom-formatting

**Reply by NgYueHong** [reply]
Thanks a lot for your help.
